packages/api/src/config/context/OrmContext.ts

```diff
@@ -38,13 +39,19 @@ export class OrmContext {
     // Register the ORM connection.
     container.register("orm", asValue(connection));
 
-    // Register a repository for each model.
-    MODELS.forEach(model => {
+    MODELS.forEach(async model => {
+      // Register a repository for each model.
       const name = createRepositoryName(model);
       const proxy = (orm: Connection) => orm.getRepository(model);
       const resolver = asFunction(proxy).singleton();
       container.register(name, resolver);
+
+      // Fire the create initial entities hook
+      if (model.createInitialEntities) {
```

Comment:
It might be best to do this in a second loop. As it is, the only repository you can be sure is prepared is your own inside of a createInitialEntities hook.

Reply:
👍